queries/flights.py

```diff
@@ -258,12 +243,18 @@ def QFlightsDepLoc(node, params, result):
 _FLIGHTS_CACHE_TTL = 600  # seconds, ttl = 10 mins
 
 # Cache for flights either departing or arriving
-_FLIGHT_CACHE = cachetools.TTLCache(maxsize=2, ttl=_FLIGHTS_CACHE_TTL)
+_FLIGHT_CACHE: cachetools.TTLCache = cachetools.TTLCache(
+    maxsize=2, ttl=_FLIGHTS_CACHE_TTL
+)
+
+# For type checking
+FlightType = Dict[str, Any]
```

Comment:
FYI, for maximum fanciness and robustness, you could put in a `TypedDict` here. A TypedDict specifies the types of values for each string key in the dict. In this case, there is a limited number of known string keys so a TypedDict would work well.

Reply:
I created a TypedDict but had to ignore a few warnings to get a clean run, as the static type checker sometimes raised false positives (e.g. thinking that a variable could still be None inside an if-statement that explicitly checked for it not being None).

Reply:
In my experience the type checkers are still primitive in this regard; I usually resort to going back from `x["y"]` to `x.get("y", "")`, where the field `y` is an `Optional[str]`; this silences the warnings.